fsspec/spec.py

```diff
@@ -427,7 +427,7 @@ def find(self, path, maxdepth=None, withdirs=False, **kwargs):
         path = self._strip_protocol(path)
         out = dict()
         detail = kwargs.pop("detail", False)
-        for path, dirs, files in self.walk(path, maxdepth, detail=True, **kwargs):
+        for path_, dirs, files in self.walk(path, maxdepth, detail=True, **kwargs):
```

Comment:
Maybe we can say just this here:
```suggestion
        for _, dirs, files in self.walk(path, maxdepth, detail=True, **kwargs):
```